spynnaker/pyNN/models/neuron/population_machine_vertex.py

```diff
@@ -92,22 +94,22 @@ class PopulationMachineVertex(
     class REGIONS(Enum):
         """Regions for populations."""
         SYSTEM = 0
-        NEURON_PARAMS = 1
-        CURRENT_SOURCE_PARAMS = 2
-        SYNAPSE_PARAMS = 3
-        POPULATION_TABLE = 4
-        SYNAPTIC_MATRIX = 5
-        SYNAPSE_DYNAMICS = 6
-        STRUCTURAL_DYNAMICS = 7
-        NEURON_RECORDING = 8
-        PROVENANCE_DATA = 9
-        PROFILING = 10
-        CONNECTOR_BUILDER = 11
-        DIRECT_MATRIX = 12
-        BIT_FIELD_FILTER = 13
-        BIT_FIELD_BUILDER = 14
-        BIT_FIELD_KEY_MAP = 15
-        RECORDING = 16
+        CORE_PARAMS = 1
```

Comment:
this is where auto() comes in useful

Reply:
Decided against as the region numbers are critical